skill-tree/src/test/java/com/RDS/skilltree/EndorsementsIntegrationTests.java

```diff
@@ -290,4 +290,96 @@ public void testAPIReturn404_OnEndorsementNotFound() {
                 .body("message", equalTo("Endorsement not found"));
     }
 
+    @Test
+    @DisplayName("Return 200 with the endorsements of a particular user given userID")
+    public void testAPIReturn200_OnEndorsementSearchByUserID() {
+        String userIDString = "f13ac7a0-76ab-4215-8bfc-2dd5d9f8ebeb";
+
+            given()
+                .when()
+                    .get("/v1/endorsements?userID=" + userIDString)
+                .then()
+                    .statusCode(200)
+                    .contentType("application/json")
+                    .body("content", everyItem(hasKey("user_id")))
+                    .body("content.user_id", everyItem(equalTo(userIDString)));
+    }
+
+    @Test
+    @DisplayName("Return 200 with the endorsements of a particular skill given skillID")
+    public void testAPIReturn200_OnEndorsementSearchBySkillID() {
+        String skillIDString = "7a6b8876-44e3-4b18-8579-79e9d4a5f0c9";
+            given()
+                .when()
+                    .get("/v1/endorsements?skillID=" + skillIDString)
+                .then()
+                    .statusCode(200)
+                    .contentType("application/json")
+                    .body("content", everyItem(hasKey("skill_id")))
+                    .body("content.skill_id", everyItem(equalTo(skillIDString)));
+    }
+
+    @Test
+    @DisplayName("Return 200 with the endorsements matching the given userID and skillID")
+    public void testAPIReturn200_OnEndorsementSearchByBothUserIDAndSkillID() {
+        String userIDString = "73e0b7c4-d128-4e53-9501-0e7f4ff5a261";
+        String skillIDString = "7a6b8876-44e3-4b18-8579-79e9d4a5f0c9";
+
+            given()
+                .when()
+                    .get("/v1/endorsements?skillID=" + skillIDString + "&userID=" + userIDString)
+                .then()
+                    .statusCode(200)
+                    .contentType("application/json")
+                    .body("content", everyItem(hasKey("skill_id")))
+                    .body("content.skill_id", everyItem(equalTo(skillIDString)))
+                    .body("content", everyItem(hasKey("user_id")))
+                    .body("content.user_id", everyItem(equalTo(userIDString)));
```

Comment:
please follow this format we have for tests
```
 Response response = given()
                .cookies(RestAPIHelper.getUserCookie())
                .contentType("application/json")
                .body(skillDRO)
                .post("/v1/skills/");

        response.then()
                .statusCode(201)
                .contentType("application/json")
                .body("name", equalTo("Go"))
                .body("type", equalTo("ATOMIC"));
                ```

Reply:
Thank you, updated the PR